spec/system/hiring_staff_can_manage_vacancy_applications_spec.rb

```diff
@@ -0,0 +1,205 @@
+require "rails_helper"
+
+RSpec.describe "Hiring staff can manage vacancy applications" do
```

Comment:
Could you add a test to make sure that no draft application is shown?

Reply:
one or two other tests i need to add, this is one didnt cross my mind